cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -71,7 +71,9 @@ function FlyoutMenu( element ) { // eslint-disable-line max-statements, no-inlin
    */
   function init() {
     var triggerClickedBinded = _triggerClicked.bind( this );
+    var triggerOverBinded = _triggerOver.bind( this );
```

Comment:
I don't think there is a need for the `Binded` suffix. I think it is implied that `this` has the same scope as other functions within the `FlyoutMenu` function.